rd-net/RdFramework.Reflection/RdFramework.Reflection.csproj

```diff
@@ -0,0 +1,71 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.0;net35;net461</TargetFrameworks>
+        <AssemblyName>JetBrains.RdFramework.Reflection</AssemblyName>
+        <RootNamespace>JetBrains.Rd.Reflection</RootNamespace>
+
+        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+        <NoWarn>1591,1570,1574</NoWarn>
+
+        <DefineConstants Condition="'$(Configuration)' == 'Debug'">JET_MODE_ASSERT</DefineConstants>
+
+        <Configurations>Debug;Release;CrossTests</Configurations>
+
+        <Platforms>AnyCPU</Platforms>
+    </PropertyGroup>
+
+
+    <PropertyGroup Label="NuGet">
+        <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+        <PackageId>JetBrains.RdFramework.Reflection</PackageId>
+        <PackageProjectUrl>https://github.com/JetBrains/rd</PackageProjectUrl>
+        <RepositoryUrl>https://github.com/JetBrains/rd</RepositoryUrl>
+        <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+        <RepositoryType>git</RepositoryType>
+        <Authors>JetBrains</Authors>
+        <Copyright>Copyright © JetBrains 2019</Copyright>
+        <PackageTags>rd reactive distributed protocol</PackageTags>
+        <Description>Plugin for JetBrains Networking library (JetBrains.RdFramework) used for defining models using regular C# for C#-C# communications</Description>
+
+
+        <IncludeSymbols>true</IncludeSymbols>
+        <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+
+        <PublishRepositoryUrl>true</PublishRepositoryUrl>
+        <EmbedUntrackedSources>true</EmbedUntrackedSources>
+
+        <PackageIcon>PackageIcon.png</PackageIcon>
+        <PackageReleaseNotes>Initial release</PackageReleaseNotes>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(Configuration)' == 'Debug'" Label="PackPdbIntoDebugNupkg">
+        <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-19367-01" PrivateAssets="all" />
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(TargetFramework)' == 'net35'">
```

Comment:
Do we really need these dependencies in this package?

Reply:
No, thanks. Updated